Core/Src/BroadcastingTask.c

```diff
@@ -13,13 +13,16 @@
 #include "Tasks.h"
 #include "radio.h"
 #include "LED.h"
+#include "config.h"
 #include <stdio.h>
 
 void BroadcastingTask(void *argument){
     
     Ethernet_Init();
+    debugprintf("Ethernet: connected to server\n\r");
 
     while (1){
+        printf("c");
```

Comment:
What is this for?

Reply:
Should it be a debugprintf?

Reply:
don't worry about it i'm gonna take it out later 👮